### PR TITLE
Publish layers 19H2

### PR DESCRIPTION
**Pull request by jsanz**
Pull request to merge into `master` the data contributions for the last months, ideally to get ready for `7.6` release of the Elastic Stack.

As a summary, this PR gathers a number of pull requests that includes the following new countries region divisions:

* Albania, Andorra, Bosnia and Herzegovina, Bulgaria, Czechia, Greece, Greenland, 
Iceland, Latvia, Liechtenstein, Lithuania, North Macedonia, Moldova, Montenegro, Romania, Serbia, Ukraine.

And improvements to:

* Italy, France, Spain, South Korea, United States

Additionally, there are some small improvements to the existing tests and helper scripts.

### CODE REVIEW

File: sources/ba/bosnia-herzegovina-territories.hjson [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      Territories Of Bosnia And Herzegovina
+    ]
+    humanReadableName: {
+      ar: أراضي البوسنة والهرسك
+      be: Тэрыторыі Босніі і Герцагавіны
+      bs: Teritorija Bosne i Hercegovine
+      ca: Territoris de Bòsnia i Hercegovina
+      cs: Území Bosny a Hercegoviny
+      de: Gebiete von Bosnien und Herzegowina
+      en: Territories of Bosnia and Herzegovina
```
Comment (nickpeihl) [inline sources/ba/bosnia-herzegovina-territories.hjson]:
```suggestion
      en: Bosnia and Herzegovina Territories
```

File: sources/kl/greenland_municipalities.hjson [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            '''
+    }
+    name: greenland_municipalities
+    legacyIds: [
+      Greenland Municipalities
+    ]
+    humanReadableName: {
+      da: Grønlandsk kommune
+      de: Kommune
+      en: Greenland municipality
```
Comment (nickpeihl) [inline sources/kl/greenland_municipalities.hjson]:
```suggestion
      en: Greenland Municipalities
```

File: sources/li/municipalities.hjson [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      bar: Vawoitungsgliedarung vo Liachtnstoa
+      be: Адміністрацыйны падзел Ліхтэнштэйна
+      be-tarask: муніцыпалітэт Ліхтэнштайну
+      ca: Municipis de Liechtenstein
+      cs: Kraje v Lichtenštejnsku
+      cy: bwrdeistref Liechtenstein
+      da: Liechtensteins kommuner
+      de: Gemeinde in Liechtenstein
+      el: Δήμοι του Λίχτενσταϊν
+      en: Liechtenstein municipalities
```
Comment (nickpeihl) [inline sources/li/municipalities.hjson]:
```suggestion
      en: Liechtenstein Municipalities
```

File: sources/lt/lithuania-counties.hjson [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      ar: مقاطعة في ليتوانيا
+      be: павет Літвы
+      bs: okruzi Litvanije
+      ca: Comtats de Lituània
+      crh: Litvaniyanıñ nahiyeleri
+      cs: Kraje v Litvě
+      da: Amter i Litauen
+      de: Bezirk in Litauen
+      el: Επαρχίες της Λιθουανίας
+      en: Counties of Lithuania
```
Comment (nickpeihl) [inline sources/lt/lithuania-counties.hjson]:
```suggestion
      en: Lithuania Counties
```

File: sources/me/montenegro-municipalities.hjson [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    humanReadableName: {
+      ar: بلدية في الجبل الأسود
+      az: Çernoqoriyanın inzibati-ərazi bölgüsü
+      be: Адміністрацыйны падзел Чарнагорыі
+      bs: Administrativna podjela Crne Gore
+      ca: Municipalitats de Montenegro
+      cs: Obce a města v Černé Hoře
+      de: Gemeinde in Montenegro
+      el: Δήμος του Μαυροβουνίου
+      en: Montenegro municipalities
```
Comment (nickpeihl) [inline sources/me/montenegro-municipalities.hjson]:
```suggestion
      en: Montenegro Municipalities
```

File: sources/rs/serbia-districts.hjson [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+      be-tarask: акруга Сэрбіі
+      bg: окръг в Сърбия
+      bs: Upravni okruzi u Srbiji
+      ca: Districte de Sèrbia
+      cs: Administrativní členění Srbska
+      cy: ardal Serbia
+      de: Verwaltungsbezirk in Serbien
+      de-ch: Verwaltungsbezirk in Serbien
+      el: Διοικητική διαίρεση της Δημοκρατίας της Σερβίας
+      en: Serbia districts
```
Comment (nickpeihl) [inline sources/rs/serbia-districts.hjson]:
```suggestion
      en: Serbia Districts
```

File: sources/ua/ukraine-regions.hjson [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+      ar: أوبلاستات أوكرانيا
+      ast: Oblast d'Ucraína
+      be: вобласць Украіны
+      be-tarask: вобласьць Украіны
+      bs: Oblasti u Ukrajini
+      ca: Oblast d'Ucraïna
+      da: Ukraines oblaster
+      de: Oblast in der Ukraine
+      el: Περιφέρεια της Ουκρανίας
+      en: Oblasts of Ukraine
```
Comment (nickpeihl) [inline sources/ua/ukraine-regions.hjson]:
```suggestion
      en: Ukraine Regions
```

**Comment by jsanz** [comment]
thanks @nickpeihl for the review!! :hugs:  since I'm not going to be around for the next few days I'd feel better if you merge the branch in case there's something else to take care of. It's also fine to me to until my return from :palm_tree: 

Regarding those old commits on this PR, I agree it's not a big deal to keep them here, rewriting the commit history looks like not worth the risk.

**Comment by nickpeihl** [comment]
@thomasneirynck @jsanz How important is it to retain the git history of the commits in this PR? It seems a previous rebase caused duplicate commits with different commit ids to appear here. I can easily squash the commits to merge, but we will lose the individual commits for each country in this PR.

**Comment by jsanz** [comment]
I would rather have duplicated commits than a full squash of the PR changes, but not strong on this at all, since most of the changes are new files and data; not too much blame to look for :sweat_smile:

**Pull request merged**